ksql-cli/src/test/java/io/confluent/ksql/cli/console/ConsoleTest.java

```diff
@@ -85,14 +86,23 @@ public void after() {
 
   @Test
   public void testPrintGenericStreamedRow() throws IOException {
-    StreamedRow row = new StreamedRow(new GenericRow(ImmutableList.of("col_1", "col_2")));
+    StreamedRow row = StreamedRow.row(new GenericRow(ImmutableList.of("col_1", "col_2")));
     terminal.printStreamedRow(row);
   }
 
   @Test
   public void testPrintErrorStreamedRow() throws IOException {
-    StreamedRow row = new StreamedRow(new FakeException());
-    terminal.printStreamedRow(row);
+    final FakeException exception = new FakeException();
+
+    terminal.printStreamedRow(StreamedRow.error(exception));
+
+    assertThat(terminal.getOutputString(), is(exception.getMessage() + "\n"));
```

Comment:
nit: `equalTo(...)` 
same below

Reply:
Not sure I follow. Are you saying change to `is(equalTo(...))` or just `equalTo(...)` - IMHO `is(...)` is also perfectly valid and less key strokes :D